tests/test_api.py

```
@@ -0,0 +1,96 @@
+            "language": "en",
+            "language_name": "English",
+            "chunks_used": 1,
+            "citations": [],
+            "processing_time": 0.1,
+            "timestamp": "2026-06-30T00:00:00Z"
+        }
+
+        # Make 31 POST requests to /query (rate limit is 30/minute)
+        # The 31st request should be rate limited
+        rate_limit_exceeded = False
+        for i in range(31):
```

Ruff (0.15.20)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)